webpack/assets/javascripts/react_app/components/Layout/layout.scss

```diff
@@ -87,6 +87,14 @@
       }
     }
   }
+
+  .visible-xs-block {
+    display: none !important;
+
+    @media (max-width: 1002px) {
+      display: block !important;
+    }
+  }
```

Comment:
I wonder why exactly `1002px` ?
basically it should be visible only on `xs` views, right?

Reply:
On second thought this actually doesn't make sense..
changed to be pf4 break point for the PageHeader Items.
If the user name is too long and the screen is too small, UI will still look broken but it's also happening in PF4 docs..
for example 
![Screenshot from 2020-08-30 19-06-31](https://user-images.githubusercontent.com/30431079/91663858-13016600-eaf4-11ea-8636-741530d8635c.png)